db/schema/transaction.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+const user = require('user.js')
+
+const transactionSchema = new mongoose.Schema({
+    uuid:String,
+    date:String,
+    items:Array,
+    tax:Number,
+    tip:Number,
+    currency:{String, default:"USD"},
+    total:Number,
+    creator:user,
+    people:Array,
+    status:{

```

Comment:
I believe we still need to assign a type attribute for status. Let's add `type: String`.

Reply:
ok i think it was here actually where they said you dont need string for enum, let me try to find it again